packages/@aws-cdk/aws-glue-alpha/lib/job-executable.ts

```diff
@@ -466,6 +485,15 @@ export interface JobExecutableConfig {
    */
   readonly extraPythonFiles?: Code[];
 
+  /**
+   * Additional Python modules that AWS Glue adds to the Python path before executing your script.
+   *
+   * @default - no extra python files specified.
+   *
+   * @see `--s3-py-modules` in https://docs.aws.amazon.com/glue/latest/dg/author-job-ray-job-parameters.html
```

Comment:
@see expects a url after the tag, and nothing else

Reply:
fixed